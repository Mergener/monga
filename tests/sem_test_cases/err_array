type floatarray = [float];

function foo(): floatarray {
    return new int[25];
}