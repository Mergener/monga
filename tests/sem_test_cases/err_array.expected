[Error] Module tests/sem_test_cases/err_array, line 4, column 24: Cannot return value of type '@array_int' in function declared with return type 'floatarray'.
Semantic errors were caught during analysis.
